docs/resources/google_compute_instance.md.erb

```diff
@@ -75,11 +75,17 @@ The following examples show how to use this InSpec audit resource.
       its('first_network_interface_type'){ should eq "one_to_one_nat" }
     end
 
+### Test that a particular compute instance label key is present
+
+    describe google_compute_instance(project: 'chef-inspec-gcp',  zone: 'us-east1-b', name: 'inspec-test-vm') do
```

Comment:
This seems to be an unrelated change to the PR

Reply:
was related to the upcoming blog post example that uses the plural resources